### PR TITLE
cmigemo: init at 1.3e

### PR DESCRIPTION
**Pull request by cohei**
cmigemo is a tool that supports Japanese incremental search with Romaji.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by cohei** [comment]
Maintainers not yet set. Who should I assign?

**Comment by doronbehar** [comment]
> Maintainers not yet set. Who should I assign?

Yourself.

**Comment by doronbehar** [comment]
I don't know why but you should strive to fix the ci errors. Before pushing you can make sure they won't fail if you'll follow:

https://github.com/NixOS/ofborg#how-does-ofborg-call-nix-instantiate

**Comment by cohei** [comment]
While building, `curl` downloads a dictionary for cmigemo to convert words. But only `fetch*` nix functions can download something in sandbox. I think that is why ci failed.

I'm trying to fix it.

**Comment by cohei** [comment]
In `installPhase`, `install` command fails only in sandbox. Why...

**Comment by cohei** [comment]
@gebner Thank you! I'll try.

**Comment by cohei** [comment]
@gebner It works. Thanks again!

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -1641,6 +1641,12 @@
     githubId = 5561189;
     name = "Cody Opel";
   };
+  cohei = {
```
Comment (gebner) [inline maintainers/maintainer-list.nix:1644]: Please make the maintainer list addition into a separate commit (`maintainers: add cohei`).

**Review by gebner — APPROVED** [review]
Looks good to me otherwise.

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (cohei) [reply maintainers/maintainer-list.nix:1644]: 👌
